### PR TITLE
Ltd 3993 embed hyperlinks

### PR DESCRIPTION
**Pull request by kevincarrogan**
### Aim

This is a small refactoring for generating PDFs.

### CODE REVIEW

File: api/letter_templates/templates/letter_templates/application_form.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {% extends "letter_templates/pdf_generation_base.html" %}
 {% load custom_tags %}
 
+{% block title %}No licence required letter{% endblock %}
+
```
Comment (saruniitr) [inline api/letter_templates/templates/letter_templates/application_form.html:5]: Is this correct? probably `Standard individual export licence`

**Review by saruniitr — APPROVED** [review]

File: api/letter_templates/templates/letter_templates/application_form.html (hunk 1 above)
Comment (kevincarrogan) [reply api/letter_templates/templates/letter_templates/application_form.html:5]: Hah. I remember you bringing this up in the PR for this before and that didn't get fixed. I'll fix it now.

Comment (kevincarrogan) [reply api/letter_templates/templates/letter_templates/application_form.html:5]: Found the original ticket. It should be "Application form".

File: api/letter_templates/helpers.py [hunk 2]
```diff
@@ -5,20 +5,11 @@
...
-from api.core.exceptions import NotFoundError
 from api.conf.settings import CSS_ROOT
 from api.letter_templates.context_generator import get_document_context
-from api.letter_templates.models import LetterTemplate
-from api.letter_templates.constants import TemplateTitles
-
-ALLOWED_TAGS = ["b", "strong", "em", "u", "h1", "h2", "h3", "h4", "h5", "h6"]
 
 
-def get_letter_template(pk):
```
Comment (currycoder) [inline api/letter_templates/helpers.py:17]: So this just wasn't used anywhere?

**Review by currycoder — APPROVED** [review]

File: api/letter_templates/helpers.py (hunk 2 above)
Comment (kevincarrogan) [reply api/letter_templates/helpers.py:17]: Unless there's some funky dynamic lookup of this function a ctfl-f suggested that it wasn't used and the tests working plus it being not covered in code coverage all point to it not being used.

**Comment by seijihg** [comment]
hmmm how this PR fixed link embedding? cant see it

**Pull request merged**